tests/Microsoft.Identity.Test.Unit.net45/CoreTests/CacheTests/CacheFallbackOperationsTests.cs

```diff
@@ -89,7 +86,8 @@ public void GetAllAdalUsersForMsal_ScopedBy_ClientIdAndEnv()
             adalUsers = CacheFallbackOperations.GetAllAdalUsersForMsal(
                 _logger,
                 _legacyCachePersistence,
-                "other_client_id");
+                "other_client_id",
+                MsalTestConstants.SovereignEnvironment);

```

Comment:
should this constant be "SovereignNetworkEnvironment" since there may be a different cache environment and that's important in these tests?